src/main/java/org/prebid/server/activity/infrastructure/creator/privacy/uscustomlogic/USCustomLogicModuleCreator.java

```diff
@@ -48,14 +48,15 @@ public USCustomLogicModuleCreator(USCustomLogicGppReaderFactory gppReaderFactory
                                       JsonLogic jsonLogic,
                                       Integer cacheTtl,
                                       Integer cacheSize,
+                                      int jitter,
```

Comment:
This should be refactored as jitter, ttl, cacheSize logically can not be part of UsCustomLogic

Reply:
Ok, I will try.